templates/extension_php-cs-fixer.dist.php

```diff
@@ -1,5 +1,17 @@
 <?php
 
+/*
+ * This file is part of the TYPO3 project.
+ *
+ * (c) 2019-2021 Benni Mack
+ *               Simon Gilli
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
```

Comment:
Why should we add this? This is copied in each extensions' directory, and actually without the doc comment it's a 3 liner. The LICENSE file does not exist for projects etc.

Reply:
Hehe, good catch. Will have to exlude it from the fixer....